docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -296,6 +296,46 @@ To enable denormals optimization in the application, the `denormals_optimization
 
 @endsphinxdirective
 
+### Sparse weights decompression
+There is a known issue that the computing power of modern hardware is growing faster than memory. Using more and more modern HW CPUs, we may never reach the theoretical peak performance of some models due to the memory bound problem. `Sparse weights decompression feature` allows to achieve greater performance on such models just due to more reasonable memory consumption.
+
+Sparse matrix is a matrix in which most of the elements are zero. To use memory more efficiently, we can store only the values of non-zero elements of such matrices in memory using special storage structures (e.g., CSC - Compressed Sparse Column or CСS - Compressed Column Storage). Using this, we can reduce the memory load and accordingly achieve higher performance on the models.
+
+`Sparse weights decompression feature` allows to pack weights for FullyConnected layers directly in the CPU plugin at the model compilation stage and store non-zero values in a special packed format. Then, during the execution of the model, the weights are unpacked and used in the computational kernel.
+
+To use this feature, the user is provided with property `sparse_weights_decompression_rate`, which can take values from the interval [0, 1]. `sparse_weights_decompression_rate` is the minimum fraction of nulls above which we use weights compression. The default value is `1`, which means the option is disabled.
+
+If the user passes an option with a value other than `1`, then during the compilation of the model for all weights of the FullyConnected layers, the number of zero elements (sparse_rate) will be calculated. If sparse_rate is higher than `sparse_weights_decompression_rate` then the weights will be packed into a special compressed format (packed).
+
+> **Example**: If user set `sparse_weights_decompression_rate` to `0.8`, then for a tensor with a calculated sparse rate of `0.9`, the weights will be packed into a special format and the weights will be decompressed at the kernel execution stage. For a tensor with a sparse rate of `0.7`, the weights will remain in memory in a dense format.
+
+#### Restrictions
+Currently, the sparse weights decompression feature is supported with the following restrictions:
+1.	Feature is only supported for FullyConnected layers.
+2.	HW must support AMX isa.
+3.	The number of input and output channels of the weights must be a multiple of 64.
+4.	The minimum sparse rate transmitted by the user cannot be less than `0.5`. If this number is less, then it will be automatically changed to `0.5`. Thus, spars decompression will only work if the sparsity is greater than or equal to `0.5`, otherwise, the forced feature is not used.
+
```

Comment:
Strange behavior. Won't be better than to limit the parameter by [0.5, 1]?

Reply:
fixed, thanks